tonic-web/src/layer.rs

```diff
@@ -4,32 +4,43 @@ use tower_layer::Layer;
 use tower_service::Service;
 
 /// Layer implementing the grpc-web protocol.
-#[derive(Debug, Clone)]
-pub struct GrpcWebLayer {
-    _priv: (),
+#[derive(Debug)]
+pub struct GrpcWebLayer<RespBody> {
+    _markers: std::marker::PhantomData<RespBody>,
 }
 
-impl GrpcWebLayer {
+impl<RespBody> Clone for GrpcWebLayer<RespBody> {
+    fn clone(&self) -> Self {
+        Self {
+            _markers: std::marker::PhantomData,
+        }
+    }
+}
+
+impl<RespBody> GrpcWebLayer<RespBody> {
```

Comment:
We tend to use `ResBody` instead of `RespBody`, so we should make that change everywhere here to follow hat we do in the rest of the project.